source/seedsource/serializers.py

```diff
@@ -74,3 +74,10 @@ class GeneratePDFSerializer(serializers.Serializer):
     tile_layers = serializers.JSONField()
     zoom = serializers.IntegerField()
     opacity = serializers.FloatField()
+
+
+class RegionSerializer(serializers.ModelSerializer):
+
```

Comment:
I don't quite see how this breaks with formatting. See [line 67](https://github.com/consbio/seedsource/pull/1/files/a323ba6e999c3910f09bb20fed1f7e2c558fa3e1#diff-95e844b349444077462ac24e80aee49dR67). I've generally followed the convention that use of a `Meta` class in Django deserves at least 1 line between `class` declaration at the `class Meta` declaration.